packages/ruby/lib/readme/metrics.rb

```diff
@@ -67,12 +68,30 @@ def process_response(response:, env:, start_time:, end_time:)
         Readme::Metrics.logger.error Errors.bad_block_message(user_info)
       elsif request.options?
         Readme::Metrics.logger.info "OPTIONS request omitted from ReadMe API logging"
+      elsif !can_filter? request, response
+        Readme::Metrics.logger.error "Request or response body MIME type isn't supported for filtering. Omitting request from ReadMe API logging"
```

Comment:
Not sure about the log level here

Reply:
Maybe this should be `warn` instead of error. I don't think it's really an error, more a result of how things are configured plus the kind of requests that are coming through.

Reply:
We should probably make the log message for the `OPTIONS` request consistent with this log level as well.